Source/Blazorise/Blazorise.csproj

```diff
@@ -39,4 +39,35 @@
     <CompilerGeneratedFilesOutputPath>__SOURCEGENERATED__</CompilerGeneratedFilesOutputPath>
   </PropertyGroup>
 
+  <!-- Note: only copy files in debug mode, we don't want automatic actions in release mode / during publishing -->
+
+  <Target Name="PreBuild - Copy javascript dependencies"
```

Comment:
Do we need all this NPM stuff in Blazorise.csproj? Can it work without it, just with dependabot.yml?

Reply:
You don't need it, it's what we use in our projects because we use unmodified versions. But you can remove it if you never use unmodified javascript versions.